app/helpers/application_helper.rb

```diff
@@ -16,7 +16,8 @@ def atom_feed_link(title, url)
   end
 
   def short_info(version)
-    escape_once(truncate(sanitize(version.info).strip, :length => 90)).html_safe
+    info = version.info.truncate(90)
+    escape_once(sanitize(info)).strip.html_safe
```

Comment:
Could not you strip before calling `sanitize`? Both `escape_once` and `sanitize` should return HTML safe strings. Calling `html_safe` again just because you are calling `strip` doesn't feel right.